Taskfile.yml

```diff
@@ -0,0 +1,128 @@
+
+  test-e2e-node-update-separate-commands:
+    desc: "Mocha E2E Node Update - Separate commands Tests"
+    cmds:
+      - cross-env MOCHA_SUITE_NAME=\"Mocha E2E Node Update - Separate commands Tests\" c8 --report-dir='coverage/e2e-node-update-separate-commands' mocha 'test/e2e/commands/separate_node_update.test.ts' --reporter-options configFile=mocha-multi-reporter.json,cmrOutput=mocha-junit-reporter+mochaFile+junit-e2e-node-update-separate-commands.xml
+
+  test-e2e-node-delete:
+    desc: "Mocha E2E Node Delete Tests"
+    cmds:
+      - cross-env MOCHA_SUITE_NAME=\"Mocha E2E Node Delete Tests\" c8 --report-dir='coverage/e2e-node-delete' mocha 'test/e2e/commands/node_delete*.test.ts' --reporter-options configFile=mocha-multi-reporter.json,cmrOutput=mocha-junit-reporter+mochaFile+junit-e2e-node-delete.xml
+
+  test-e2e-node-delete-separate-commands:
+    desc: "Mocha E2E Node Delete - Separate commands Tests"
+    cmds:
+      - cross-env MOCHA_SUITE_NAME=\"Mocha E2E Node Delete - Separate commands Tests\" c8 --report-dir='coverage/e2e-node-delete-separate-commands' mocha 'test/e2e/commands/separate_node_delete*.test.ts' --reporter-options configFile=mocha-multi-reporter.json,cmrOutput=mocha-junit-reporter+mochaFile+junit-e2e-node-delete-separate-commands.xml
+
+  test-e2e-node-upgrade:
+    desc: "Mocha E2E Node Upgrade Tests"
+    cmds:
+      - cross-env MOCHA_SUITE_NAME=\"Mocha E2E Node Upgrade Tests\" c8 --report-dir='coverage/e2e-node-upgrade' mocha 'test/e2e/commands/node_upgrade*.test.ts' --reporter-options configFile=mocha-multi-reporter.json,cmrOutput=mocha-junit-reporter+mochaFile+junit-e2e-node-upgrade.xml
+
+  test-e2e-relay:
+    desc: "Mocha E2E Relay Tests"
+    cmds:
+      - cross-env MOCHA_SUITE_NAME=\"Mocha E2E Relay Tests\" c8 --report-dir='coverage/e2e-relay' mocha 'test/e2e/commands/relay.test.ts' --reporter-options configFile=mocha-multi-reporter.json,cmrOutput=mocha-junit-reporter+mochaFile+junit-e2e-relay.xml
+
+  test-example-taskfile:
+    desc: "Test Example Taskfile"
+    cmds:
+      - task --taskfile examples/custom-network-config/Taskfile.yml -d . default
+
+  # ---------- CLI Execution commands ----------
+
+  solo-test:
+    cmds:
+      - node --no-deprecation --no-warnings --loader ts-node/esm solo.ts
+
+  solo:
+    cmds:
+      - node --no-deprecation --no-warnings dist/solo.js
+
+  # ---------- Lint and format tasks ----------
+
+  check:
+    cmds:
+      - remark . --quiet --frail && eslint . --ignore-pattern 'docs/*' --ignore-pattern 'dist/*'; cd docs; jsdoc -c jsdoc.conf.json && tsc
+
+  format:
+    cmds:
+      - remark . --quiet --frail --output && eslint --fix . --ignore-pattern 'docs/*' --ignore-pattern 'dist/*'
+
+  # Setup, build, and miscellaneous tasks
+
+  test-setup:
+    cmds:
+      - ./test/e2e/setup-e2e.sh
+
+  build:
```

Comment:
I would like to be able to run `task build` or `task format` without having to type `npm run task build`.

```
❯ task build
task: [build] swc src version.ts solo.ts -d dist && node resources/post-build-script.js
"swc": executable file not found in $PATH
task: Failed to run task "build": exit status 127
❯ task format
task: [format] remark . --quiet --frail --output && eslint --fix . --ignore-pattern 'docs/*' --ignore-pattern 'dist/*'
"remark": executable file not found in $PATH
task: Failed to run task "format": exit status 127
```